src/main/java/org/jitterbit/connector/dropbox/activities/ProcessFileActivity.java

```diff
@@ -216,6 +281,29 @@ private void init() {
     model.schemaContentType = org.jitterbit.connector.sdk.metadata.SchemaMetaData.SchemaContentType.XML;
     model.filename = obj.getObjectName() + ".xml";
 
+    model = new DiscoverableObjectModel();
+    obj = new DiscoverableObject();
+    obj.setObjectName("root")
+        .setObjectType("xsd")
+        .setObjectDesc("Multiple schemas");
+    model.obj = obj;
+    model.path = "sample-multiple-xsd";
+    model.ext = "xsd";
+    model.schemaContentType = SchemaContentType.XSD;
+    model.filename = obj.getObjectName() + ".xsd";
+    objects.add(model);
+
+    model = new DiscoverableObjectModel();
+    obj = new DiscoverableObject();
+    obj.setObjectName("root2")
+        .setObjectType("xsd")
+        .setObjectDesc("Multiple schemas as Xsd(4 XML Schemas)");
```

Comment:
`.setObjectDesc("Multiple schemas as XSDs (4 XML Schemas)");`

Reply:
just a new instance - not discarding